### PR TITLE
GHI-271 Implemented getOwners with integration test

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- getOwners result decoding
- integration test

### CODE REVIEW

File: MultisigWallet/MultisigWalletDomainModel/EthereumContractProxy.swift [hunk 1]
```diff
@@ -19,17 +19,37 @@ public class EthereumContractProxy {
     }
 
     func decodeUInt(_ value: Data) -> BigUInt {
-        let bigEndianValue = value.prefix(32)
+        let bigEndianValue = value[value.startIndex..<value.startIndex + 32]
```
Comment (sche) [inline MultisigWallet/MultisigWalletDomainModel/EthereumContractProxy.swift]: does it produce different results?

File: safe/safe/IntegrationTests/InfuraEthereumNodeServiceTests.swift [hunk 2]
```diff
@@ -105,4 +105,15 @@ class InfuraEthereumNodeServiceTests: BlockchainIntegrationTest {
         XCTAssertEqual(balance, TokenInt("20000000000000000000000"))
     }
 
+    func test_safe_getOwners() throws {
+        DomainRegistry.put(service: service, for: EthereumNodeDomainService.self)
+        let proxy = SafeOwnerManagerContractProxy(Address("0x092CC1854399ADc38Dad4f846E369C40D0a40307"))
```
Comment (sche) [inline safe/safe/IntegrationTests/InfuraEthereumNodeServiceTests.swift]: Should we put it into test stubs?

File: safe/safe/IntegrationTests/InfuraEthereumNodeServiceTests.swift [hunk 3]
```diff
@@ -105,4 +105,15 @@ class InfuraEthereumNodeServiceTests: BlockchainIntegrationTest {
...
+        let expected = ["0xd06ab3c0d8094791f8f3bdb6b66cb82a68b6d846",
+                        "0xb952005d631d4892430144a2d5850b1cd0efc981",
+                        "0x41b152984f80c4017d3640662727c263e2073780"].map { Address($0) }
+        let owners = try proxy.getOwners()
+        print(owners)
```
Comment (sche) [inline safe/safe/IntegrationTests/InfuraEthereumNodeServiceTests.swift]: please remove

**Comment by sche** [comment]
Yay!

**Review by sche — APPROVED** [review]

File: MultisigWallet/MultisigWalletDomainModel/EthereumContractProxy.swift (hunk 1 above)
Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletDomainModel/EthereumContractProxy.swift]: yes, it turns out that suffix and prefix are working on the underlying collection and not on a slice of a collection.. I may double-check with these to use prefix-suffixes again, but it is this way it worked yesterday

File: safe/safe/IntegrationTests/InfuraEthereumNodeServiceTests.swift (hunk 3 above)
Comment (DmitryBespalov) [reply safe/safe/IntegrationTests/InfuraEthereumNodeServiceTests.swift]: ok

**Pull request merged**